tests/TestRunner/TestRunner.py

```diff
@@ -279,7 +280,14 @@ def run_simulation_test(basedir, options):
     test_dir = td.get_current_test_dir()
     if options.seed is not None:
         pargs.append('-s')
-        pargs.append("{}".format(int(options.seed, 0)))
+        seed = int(options.seed, 0)
+        if options.test_number:
+            idx = int(options.test_number)
+            random.seed(seed)
+            for i in range(idx):
+                # Python does not have a limit for integers
+                seed = random.randint(0, 2**32 - 1)
```

Comment:
```suggestion
                seed = random.randint(1, 2**32 - 1)
```

Simulation rejects seed == 0 ... for some reason that I've forgotten.